src/txn/txn_rollback_to_stable.c

```diff
@@ -148,6 +191,31 @@ __rollback_row_add_update(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW *rip,
     return (ret);
 }
 
+/*
+ * __rollback_col_ondisk_fixup_key --
+ *     Allocate tombstone and calls function add update to head of insert list.
+ */
+static int
+__rollback_col_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_PAGE *page, WT_REF *ref, WT_COL *cip,
```

Comment:
The page is part of the `ref`. It can be accessed via `ref->page`. There is no need to pass both.

Reply:
Would that also apply to the above callers that use page?

Reply:
Yes. When the rollback to stable operating on a `ref` it must have a `ref->page`. So all the callers also can be changed to use it.

Reply:
Okay no worries